pkg/oci/signed/image.go

```diff
@@ -44,3 +46,8 @@ func (*image) Signatures() (oci.Signatures, error) {
 func (*image) Attestations() (oci.Signatures, error) {
 	return empty.Signatures(), nil
 }
+
+// Attestations implements oci.SignedImage
+func (*image) Attachment(name string) (oci.File, error) {
+	return nil, errors.New("unimplemented")
+}
```

Comment:
It would be good to sketch out the `ociremote` forms of this, and use it in the read paths to validate this and see how the pieces will tie together.

We can follow up with things like `mutate.AttachFile` and `ociremote.WriteAttachment` for the "write" paths.

Reply:
whoops looking in the wrong place.  This is `signed`, but `ociremote` is implemented.